.github/workflows/build.yml

```diff
@@ -26,13 +19,11 @@ jobs:
       with:
         global-json-file: ./global.json
 
-    - name: Build ${{matrix.Game}}
-      run: ./Scripts/build.ps1 ${{matrix.Game}}
```

Comment:
Pretty sure you've omitted something necessary :P

Reply:
Oops...

Reply:
Restored in d3ed00f8eb31aa6aa5ec9f8a21a5d5e8b3c8761f